arelle/plugin/validate/ESEF/ESEF_Current/DTS.py

```diff
@@ -278,6 +315,12 @@ def lc3wordAdjust(word: str) -> str:
                 if linkEltName == "calculationLink":
                     val.hasExtensionCal = True
                     linkbasesFound.add(linkEltName)
+                    if esefTaxonomyYear >= 2024:
```

Comment:
In an offline conversation Brett raised the point that while this validation change is motivated by the 2024 IFRS taxonomy, the reporting manual doesn't specify that this only applies to the 2024 ESEF taxonomy. The actual choice of summation arcrole should be driven by the disclosure system year/reporting manual version and not the taxonomy year.

Reply:
Yes, Brett is right.
After a careful reading of the ESEF Reporting Manual, we noticed that it is immediately applicable right after the publication of the updated ESEF Reporting Manual.
Thus, we will change the code to make it depend on the disclosure system year (2024). Implicitly,  disclosure system year 2024 corresponds to the ESEF Reporting Manual, version 2024.